### ISSUE TITLE
Omnisharp server not running

### ISSUE THREAD

**Issue by Verge300** [OmniSharp/omnisharp-vscode#5229]
- Issue Type: `Bug`
- Extension Name: `csharp`
- Extension Version: `1.25.0`
- OS Version: `Windows_NT x64 10.0.19042`
- VS Code version: `1.67.2`

Hi no matter what i do my omnisharp server for my c# extension will not start up what can i do?

**Comment by dfibuch** [comment]
I've started getting the same problem today but cannot downgrade Omnisharp because the older version is not compatible with latest version of VSC. The error I'm seeing from Omnisharp is this

`[ERROR] Error: Found dotnet version 5.0.408. Minimum required version is 6.0.100.`

I'm running in a dev container on Linux so I just added this command to my Dockerfile and it now works

`apt-get -y install -y dotnet-sdk-6.0`

Or you can install it manually here https://dotnet.microsoft.com/en-us/download/dotnet/thank-you/sdk-6.0.300-windows-x64-installer

**Comment by filipw** [comment]
Please see the announcement https://github.com/OmniSharp/omnisharp-vscode#the-c-extension-no-longer-ships-with-an-included-mono--msbuild-tools

and the related issue https://github.com/OmniSharp/omnisharp-vscode/issues/5120

TL;DR
- ensure .NET SDK 6 is installed
- for .NET Core (and newer) development this should be enough
- for Unity / .NET Framework development, set `"omnisharp.useModernNet" : false` and ensure global MSBuild is installed on your machine - either using VS 2022 Build Tools (Windows) or using latest [Mono Preview](https://www.mono-project.com/download/preview/) (*nix)
